src/app/designationEditButton/designationEditButton.component.js

```diff
@@ -17,7 +17,7 @@ class DesignationEditButtonController {
 
   $onInit() {
     if(this.designationNumber && includes([Roles.identified, Roles.registered], this.sessionService.getRole())){
-      this.designationEditorService.getContent(this.designationNumber, this.campaignPage).then(() => {
+      this.designationEditorService.checkPermission(this.designationNumber, this.campaignPage).then(() => {
```

Comment:
Because a failed response code is normal behavior if the user doesn't have access to edit, I don't think that should be logged to rollbar.